public/main.js

```diff
@@ -0,0 +1,200 @@
+  document.getElementById('cell-1').addEventListener('click', () => {
+    if (oLog.includes(1) === false && xLog.includes(1) === false) {
+    placeMark(1);
+    winChecker();
+    } else {alert('Space taken')};
+  }); 
+  document.getElementById('cell-2').addEventListener('click', () => {
+    if (oLog.includes(2) === false && xLog.includes(2) === false) {
+      placeMark(2);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+  document.getElementById('cell-3').addEventListener('click', () => {
+    if (oLog.includes(3) === false && xLog.includes(3) === false) {
+      placeMark(3);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+  document.getElementById('cell-4').addEventListener('click', () => {
+    if (oLog.includes(4) === false && xLog.includes(4) === false) {
+      placeMark(4);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+  document.getElementById('cell-5').addEventListener('click', () => {
+    if (oLog.includes(5) === false && xLog.includes(5) === false) {
+      placeMark(5);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+  document.getElementById('cell-6').addEventListener('click', () => {
+    if (oLog.includes(6) === false && xLog.includes(6) === false) {
+      placeMark(6);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+  document.getElementById('cell-7').addEventListener('click', () => {
+    if (oLog.includes(7) === false && xLog.includes(7) === false) {
+      placeMark(7);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+  document.getElementById('cell-8').addEventListener('click', () => {
+    if (oLog.includes(8) === false && xLog.includes(8) === false) {
+      placeMark(8);
+      winChecker();
+      } else {alert('Space taken')};
+  });
+}
+// | - - - win check functions - - - |
+function winChecker () {
+  for (let subCombo of winCombo) {
+    let xCount = 0; // count of 3 signifies win 
+    let oCount = 0;
+    for (let num of subCombo) {
+      if (xLog.includes(num)) {
+        xCount = xCount + 1;
+      } else if (oLog.includes(num)){
+        oCount = oCount + 1; 
+      }
+    }
+    if (xCount == 3) {
+      gameStatus.textContent = playerNameX + " wins!";
+      showWin(subCombo);
+      setTimeout(() => {location.reload(); }, 1500)
+    } else if (oCount == 3) {
+      gameStatus.textContent = playerNameO + " wins!";
+      setTimeout(() => {location.reload(); }, 1500);
+      showWin(subCombo);
+    } else if (moveCount.length === 9) {
+      gameStatus.textContent = "It's a draw";
+      setTimeout(() => {location.reload(); }, 1500);
+    } 
+  }
+}
+function showWin (winningArray) {
+  for (let i = 0; i < winningArray.length; i++) {
+    document.getElementById('cell-' + winningArray[i]).style.background = "red"
+  }
+}
+// | - - - place mark on board & change player - - - |
+function placeMark(numCell) {
+  if (playerO) {
+    document.getElementById('cell-' + numCell).textContent = "0";
```

Comment:
Here's something that is confusing the program - you've got the text content set to "0" (zero), when it should be "O" (the letter O)